### DISCUSSION TITLE
Animate cursor movement / screen scrolling

### DISCUSSION THREAD

**Discussion by kik4444** [helix-editor/helix#11728] [Ideas]
Are there any plans to implement animating the cursor movement and screen scrolling similar to Neovim's [mini.animate](https://github.com/echasnovski/mini.animate) plugin

The closest existing issue I could find related to this is #1671 but that one only focuses on smooth scrolling as far as I can tell without the cursor movement.

**Comment by Keshav-writes-code** [comment]
were you able to find something for this?
